pyhifiberry/audiocontrol2sio.py

```diff
@@ -0,0 +1,141 @@
+"""Socketio based API to Hifiberry OS - audiocontrol2"""
+import logging
+
+from socketio import AsyncClient, AsyncClientNamespace
+
+from .consts import DEFAULT_HOST, DEFAULT_PORT
+
+_LOGGER = logging.getLogger(__name__)
+
+class Audiocontrol2SIO:
+
+    def __init__(self, sio) -> None:
+        self.connected = False
+        self.metadata = Metadata(sio)
+        self.player = Player(sio)
+        self.volume = Volume(sio)
+        self._sio = sio
+
```

Comment:
add `self.base_url = f"http://{host}:{port}"`

Reply:
Hi @willholdoway, I think these changes are not necessary. It is not used on your (HA hifiberry) side.

Reply:
@willholdoway do you agree?